draft-ietf-quic-transport.md

```diff
@@ -2960,8 +2960,8 @@ data to a peer.
 
 ~~~
        o
-       | Application Open
-       | Open Paired Stream (bidirectional)
+       | Open Stream
+       | Open Bidirectional Receive Stream
```

Comment:
You have the directions flipped (should be Send here and Receive below). Admittedly pedantic, but how about "Open Bidirectional Stream (Sending part)"? Calling it a "Bidirectional Send Stream" sounds silly.

Reply:
@janaiyengar do you have any suggestions for clarifying this?

Reply:
Agreed the current text is confusing.  Open Bidirectional Stream might be clearer.